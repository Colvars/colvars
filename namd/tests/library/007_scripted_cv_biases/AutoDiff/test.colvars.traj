# step         d                     fa_d                   o                                                                                                 
           0    1.42467449615693e+01  1.87662751921536e+03  (  9.99999748638998e-01 ,  6.84042724576832e-04 ,  5.12508171643214e-05 , -1.79390207161565e-04 ) 
           2    1.45977275439175e+01  1.70113622804125e+03  (  9.99985973761768e-01 ,  5.11485913744026e-03 , -3.32731800417789e-05 , -1.37455033645004e-03 ) 
           4    1.50626713340646e+01  1.46866433296768e+03  (  9.99843158592016e-01 ,  1.70395625507908e-02 , -1.43866916456290e-04 , -4.82605709972753e-03 ) 
           6    1.56168085965048e+01  1.19159570174758e+03  (  9.99321288192724e-01 ,  3.53760784411667e-02 , -2.78427047992860e-04 , -1.02673520119492e-02 ) 
           8    1.62291266298068e+01  8.85436685096593e+02  (  9.98133542942867e-01 ,  5.85622743061378e-02 , -4.51475701577846e-04 , -1.73114600800356e-02 ) 
          10    1.68643388381563e+01  5.67830580921871e+02  (  9.96054934943978e-01 ,  8.49744253292275e-02 , -6.83720355648298e-04 , -2.55625925952464e-02 ) 
          12    1.74846635628575e+01  2.57668218571256e+02  (  9.92972839557288e-01 ,  1.13139164123728e-01 , -9.51259111211344e-04 , -3.46924278340861e-02 ) 
          14    1.80530624332059e+01 -2.65312166029723e+01  (  9.88977962066854e-01 ,  1.41311479837272e-01 , -1.14108784183725e-03 , -4.41854515743639e-02 ) 
          16    1.85372200961765e+01 -2.68610048088240e+02  (  9.84381327967116e-01 ,  1.67770992125046e-01 , -1.20902908935363e-03 , -5.33369815394103e-02 ) 
          18    1.89124918771857e+01 -4.56245938592827e+02  (  9.79545563657221e-01 ,  1.91522263357025e-01 , -1.22366955206206e-03 , -6.17107283289307e-02 ) 
          20    1.91635808809730e+01 -5.81790440486497e+02  (  9.74600371909626e-01 ,  2.12939348547346e-01 , -1.24954463323043e-03 , -6.93497480317004e-02 ) 
# step         ds                    fa_ds                  vec                                                                                                                       fa_vec                                                                                                                   
          20    1.42467449615693e+01  0.00000000000000e+00  (  1.42467449615693e+01 ,  9.99999748638998e-01 ,  6.84042724576832e-04 ,  5.12508171643214e-05 , -1.79390207161565e-04 ) (  1.87662751921535e+03 ,  0.00000000000000e+00 ,  1.00000000000000e+05 ,  0.00000000000000e+00 ,  0.00000000000000e+00 )
          22    1.45977275439175e+01  0.00000000000000e+00  (  1.45977275439175e+01 ,  9.99985973761768e-01 ,  5.11485913744027e-03 , -3.32731800417788e-05 , -1.37455033645004e-03 ) (  1.70113622804125e+03 ,  0.00000000000000e+00 ,  1.00000000000000e+05 ,  0.00000000000000e+00 ,  0.00000000000000e+00 )
          24    1.50626713340646e+01  0.00000000000000e+00  (  1.50626713340647e+01 ,  9.99843158592016e-01 ,  1.70395625507908e-02 , -1.43866916456290e-04 , -4.82605709972753e-03 ) (  1.46866433296765e+03 ,  0.00000000000000e+00 ,  1.00000000000000e+05 ,  0.00000000000000e+00 ,  0.00000000000000e+00 )
          26    1.56168085965049e+01  0.00000000000000e+00  (  1.56168085965048e+01 ,  9.99321288192724e-01 ,  3.53760784411667e-02 , -2.78427047992838e-04 , -1.02673520119492e-02 ) (  1.19159570174760e+03 ,  0.00000000000000e+00 ,  1.00000000000000e+05 ,  0.00000000000000e+00 ,  0.00000000000000e+00 )
          28    1.62291266298068e+01  0.00000000000000e+00  (  1.62291266298068e+01 ,  9.98133542942867e-01 ,  5.85622743061378e-02 , -4.51475701577845e-04 , -1.73114600800356e-02 ) (  8.85436685096600e+02 ,  0.00000000000000e+00 ,  1.00000000000000e+05 ,  0.00000000000000e+00 ,  0.00000000000000e+00 )
          30    1.68643388381563e+01  0.00000000000000e+00  (  1.68643388381563e+01 ,  9.96054934943978e-01 ,  8.49744253292276e-02 , -6.83720355648297e-04 , -2.55625925952464e-02 ) (  5.67830580921850e+02 ,  0.00000000000000e+00 ,  1.00000000000000e+05 ,  0.00000000000000e+00 ,  0.00000000000000e+00 )
          32    1.74846635628575e+01  0.00000000000000e+00  (  1.74846635628575e+01 ,  9.92972839557288e-01 ,  1.13139164123728e-01 , -9.51259111211365e-04 , -3.46924278340861e-02 ) (  2.57668218571251e+02 ,  0.00000000000000e+00 ,  1.00000000000000e+05 ,  0.00000000000000e+00 ,  0.00000000000000e+00 )
          34    1.80530624332059e+01  0.00000000000000e+00  (  1.80530624332059e+01 ,  9.88977962066854e-01 ,  1.41311479837272e-01 , -1.14108784183723e-03 , -4.41854515743639e-02 ) ( -2.65312166029492e+01 ,  0.00000000000000e+00 ,  1.00000000000000e+05 ,  0.00000000000000e+00 ,  0.00000000000000e+00 )
          36    1.85372200961765e+01  0.00000000000000e+00  (  1.85372200961765e+01 ,  9.84381327967116e-01 ,  1.67770992125046e-01 , -1.20902908935363e-03 , -5.33369815394102e-02 ) ( -2.68610048088250e+02 ,  0.00000000000000e+00 ,  1.00000000000000e+05 ,  0.00000000000000e+00 ,  0.00000000000000e+00 )
          38    1.89124918771856e+01  0.00000000000000e+00  (  1.89124918771856e+01 ,  9.79545563657221e-01 ,  1.91522263357025e-01 , -1.22366955206212e-03 , -6.17107283289307e-02 ) ( -4.56245938592801e+02 ,  0.00000000000000e+00 ,  1.00000000000000e+05 ,  0.00000000000000e+00 ,  0.00000000000000e+00 )
          40    1.91635808809730e+01  0.00000000000000e+00  (  1.91635808809730e+01 ,  9.74600371909626e-01 ,  2.12939348547346e-01 , -1.24954463323041e-03 , -6.93497480317004e-02 ) ( -5.81790440486500e+02 ,  0.00000000000000e+00 ,  1.00000000000000e+05 ,  0.00000000000000e+00 ,  0.00000000000000e+00 )
