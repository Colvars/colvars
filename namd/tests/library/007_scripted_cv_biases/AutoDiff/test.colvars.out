colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:  https://dx.doi.org/10.1080/00268976.2013.813594
colvars: in any publication based on this calculation.
colvars: SMP parallelism is enabled; if needed, use "smp off" to override this.
colvars: This version was built with the C++11 standard or higher.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "unscripted.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 2
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # scriptedColvarForces = on
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "d"
colvars:   Initializing a new "distance" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group1".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 12.011, total charge = -0.1.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group2".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1000
colvars:   Reading legacy options upperWall and upperWallConstant: consider using a harmonicWalls restraint (caution: force constant would then be scaled by width^2).
colvars:   # upperWall = 1000
colvars:   Generating a new harmonicWalls bias for compatibility purposes.
colvars:   # hardLowerBoundary = on [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "o"
colvars:   Initializing a new "orientation" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 104 atoms requested: total mass = 727.821, total charge = 6.70552e-08.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     Warning: beginning from 2019-11-26 the XYZ file reader assumes Angstrom units.
colvars:     Centering the reference coordinates: it is assumed that each atom is the closest periodic image to the center of geometry.
colvars:     # closestToQuaternion = ( 1 , 0 , 0 , 0 ) [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonic" instance.
colvars:   # name = "harmonic1" [default]
colvars:   # colvars = { d }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 0 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # centers = { 18 }
colvars:   # targetCenters = { 18 } [default]
colvars:   # outputCenters = off [default]
colvars:   # forceConstant = 500
colvars:   # targetForceConstant = -1 [default]
colvars:   The force constant for colvar "d" will be rescaled to 500 according to the specified width (1).
colvars: ----------------------------------------------------------------------
colvars:   User forces script will be run at each bias update.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 2 [default]
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # scriptedColvarForces = on [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonicwalls" instance.
colvars:   # name = "dw"
colvars:   # colvars = { d }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 0 [default]
colvars:   # timeStepFactor = 1
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # forceConstant = 1 [default]
colvars:   # targetForceConstant = -1 [default]
colvars:   # lowerWalls = { 0 } [default]
colvars:   Lower walls were not provided.
colvars:   # upperWalls = { 1000 }
colvars:   # upperWallConstant = 1
colvars:   The upper wall force constant for colvar "d" will be rescaled to 1 according to the specified width (1).
colvars: ----------------------------------------------------------------------
colvars:   User forces script will be run at each bias update.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (104 atoms).
colvars: updating group data (2 scalable groups, 2 atoms in total).
colvars: Re-initialized atom group for variable "d":0/0. 1 atoms: total mass = 12.011, total charge = -0.1.
colvars: Re-initialized atom group for variable "d":0/1. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "o":0/0. 104 atoms: total mass = 727.821, total charge = 6.70552e-08.
colvars: The final output state file will be "test.colvars.state".
colvars: Opening trajectory file "test.colvars.traj".
colvars: Redefining the Tcl "cv" command to the new script interface.
colvars: Updating NAMD interface:
colvars: updating atomic data (104 atoms).
colvars: updating group data (2 scalable groups, 2 atoms in total).
colvars: Re-initialized atom group for variable "d":0/0. 1 atoms: total mass = 12.011, total charge = -0.1.
colvars: Re-initialized atom group for variable "d":0/1. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "o":0/0. 104 atoms: total mass = 727.821, total charge = 6.70552e-08.
colvars: The final output state file will be "test.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
colvars: Resetting the Collective Variables module.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration from file "scripted.in":
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 2
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # scriptedColvarForces = on
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "ds"
colvars:   Initializing a new "distance" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "group1".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group1".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group1" defined with 1 atoms requested: total mass = 12.011, total charge = -0.1.
colvars:       Initializing atom group "group2".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       Enabling scalable calculation for group "group2".
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "group2" defined with 1 atoms requested: total mass = 12.011, total charge = 0.07.
colvars:     # oneSiteSystemForce = off [default]
colvars:     # oneSiteTotalForce = off [default]
colvars:   All components initialized.
colvars:   This colvar uses scripted function "Id".
colvars:   # scriptedFunctionType = "scalar" [default]
colvars:   Expecting colvar value of type scalar number
colvars:   # period = 0 [default]
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1000
colvars:   Reading legacy options upperWall and upperWallConstant: consider using a harmonicWalls restraint (caution: force constant would then be scaled by width^2).
colvars:   # upperWall = 1000
colvars:   Generating a new harmonicWalls bias for compatibility purposes.
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "vec"
colvars:   Initializing a new "cartesian" component.
colvars:     # name = "cvc2"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 2 atoms requested: total mass = 24.022, total charge = -0.03.
colvars:     # useX = on [default]
colvars:     # useY = on [default]
colvars:     # useZ = on [default]
colvars:   Initializing a new "orientation" component.
colvars:     # name = "cvc1"
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:       Initializing atom group "atoms".
colvars:       # name = "" [default]
colvars:       # centerReference = off [default]
colvars:       # rotateReference = off [default]
colvars:       # atomsOfGroup = "" [default]
colvars:       # indexGroup = "" [default]
colvars:       # psfSegID =  [default]
colvars:       # atomsFile = "" [default]
colvars:       # dummyAtom = ( 0 , 0 , 0 ) [default]
colvars:       # enableForces = on [default]
colvars:       # enableFitGradients = on [default]
colvars:       # printAtomIDs = off [default]
colvars:       Atom group "atoms" defined with 104 atoms requested: total mass = 727.821, total charge = 6.70552e-08.
colvars:     # refPositions =  [default]
colvars:     # refPositionsFile = "../Common/da.xyz"
colvars:     # refPositionsCol = "" [default]
colvars:     Centering the reference coordinates: it is assumed that each atom is the closest periodic image to the center of geometry.
colvars:     # closestToQuaternion = ( 1 , 0 , 0 , 0 ) [default]
colvars:   All components initialized.
colvars:   This colvar uses scripted function "vector".
colvars:   # scriptedFunctionType = "vector"
colvars:   Expecting colvar value of type n-dimensional vector
colvars:   # scriptedFunctionVectorSize = 5
colvars:   Sorted list of components for this scripted colvar:
colvars:   1 cvc1
colvars:   2 cvc2
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = on
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars:   User forces script will be run at each bias update.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 0 in total.
colvars: # units = "" [default]
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 2 [default]
colvars: # colvarsRestartFrequency = 0 [default]
colvars: # scriptedColvarForces = on [default]
colvars: # scriptingAfterBiases = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 2 in total.
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new "harmonicwalls" instance.
colvars:   # name = "dsw"
colvars:   # colvars = { ds }
colvars:   # stepZeroData = off [default]
colvars:   # outputEnergy = off [default]
colvars:   # outputFreq = 0 [default]
colvars:   # timeStepFactor = 1
colvars:   # writeTISamples = off [default]
colvars:   # writeTIPMF = off [default]
colvars:   # forceConstant = 1 [default]
colvars:   # targetForceConstant = -1 [default]
colvars:   # lowerWalls = { 0 } [default]
colvars:   Lower walls were not provided.
colvars:   # upperWalls = { 1000 }
colvars:   # upperWallConstant = 1
colvars:   The upper wall force constant for colvar "ds" will be rescaled to 1 according to the specified width (1).
colvars: ----------------------------------------------------------------------
colvars:   User forces script will be run at each bias update.
colvars: ----------------------------------------------------------------------
colvars: Collective variables biases initialized, 1 in total.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Updating NAMD interface:
colvars: updating atomic data (104 atoms).
colvars: updating group data (2 scalable groups, 2 atoms in total).
colvars: Saving collective variables state to "test.colvars.state".
